modules/store/src/tokens.ts

```diff
@@ -5,6 +5,9 @@ export const _INITIAL_STATE = new InjectionToken(
   '@ngrx/store Internal Initial State'
 );
 export const INITIAL_STATE = new InjectionToken('@ngrx/store Initial State');
+export const MOCK_SELECTORS = new InjectionToken(
```

Comment:
Could we move this to the testing package?

Reply:
Moved in ed7cc49.